src/librustdoc/clean/auto_trait.rs

```diff
@@ -437,7 +419,7 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
     // K', we use the dedicated syntax 'T: Fn() -> K'
     // * We explicitly add a '?Sized' bound if we didn't find any 'Sized' predicates for a type
     fn param_env_to_generics(
-        &self,
+        &mut self,
```

Comment:
What is requiring it to be `mut` here? Can't find out just from reading the changes...

Reply:
`p.clean(self.cx)` on line 473 takes `&mut DocContext`.